frontend/src/lib/components/PropertyKeyInfo.scss

```diff
@@ -3,3 +3,13 @@
 .ant-popover.property-key-info-tooltip {
     max-width: 300px;
 }
+
+.w-24 {
+    width: 24px !important;
+    height: 24px !important;
+}
+
+.align-item-center {
+    display: flex !important;
+    align-items: center;
+}
```

Comment:
We have a bunch of utility classes that you can use rather than creating custom ones for this use case. Check out storybook for some examples https://storybook.posthog.net/?path=/docs/lemon-ui-utilities--flex

Reply:
They mostly follow Tailwind so you can use [their docs](https://tailwindcss.com/docs/width) for help as well

Reply:
If the !important is "important" then we should be updating the `property-key-info` class ideally rather than adding in hard overrides this way